examples/run_detection.py

```diff
@@ -88,5 +92,53 @@ def main(argv):
         explanation.save(output, Path(args.image_path).stem)
 
 
+def explain_black_box(args):
+    """
+    Black-box scenario.
+    """
+
+    # Create ov.Model
+    model: ov.Model
+    model = ov.Core().read_model(args.model_path)
+
+    # Create explainer object
+    explainer = xai.Explainer(
+        model=model,
+        task=xai.Task.DETECTION,
+        preprocess_fn=preprocess_fn,
+        postprocess_fn=postprocess_fn,
+        explain_mode=ExplainMode.BLACKBOX,  # defaults to AUTO
+    )
+
+    # Prepare input image and explanation parameters, can be different for each explain call
+    image = cv2.imread(args.image_path)
+
+    # Generate explanation
+    explanation = explainer(
+        image,
+        targets=[0],  # target boxes to explain
+        overlay=True,
+        preset=Preset.SPEED,
+    )
+
+    logger.info(
+        f"Generated {len(explanation.saliency_map)} detection "
+        f"saliency maps of layout {explanation.layout} with shape {explanation.shape}."
+    )
+
+    # Save saliency maps for visual inspection
+    if args.output is not None:
+        output = Path(args.output) / "detection_black_box"
+        explanation.save(output, Path(args.image_path).stem)
```

Comment:
```suggestion
        explanation.save(output, f"{Path(args.image_path).stem}_")
```
We put a responsibility to provide the underscore after the prefix in naming to a user after the latest `.save` changes

Reply:
Good catch